src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -159,9 +160,19 @@ export class SbbDatepicker implements ComponentInterface {
     );
   }
 
+  @Listen('datepicker-control-registered')
+  public onInputPropertiesChange(): void {
```

Comment:
I think method does not have to be public?

Reply:
You're right, I just assumed it would throw errors when using Listen with private methods, turns out it does not 😄. Fixing this right away